src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/CpsPackageReferenceProjectProvider.cs

```diff
@@ -62,32 +67,30 @@ public bool TryCreateNuGetProject(EnvDTE.Project dteProject, ProjectSystemProvid
             {
                 return false;
             }
-            var restoreProjectStyle = string.Empty;
-            var targetFramework = string.Empty;
-            var targetFrameworks = string.Empty;
-
-            var hasRestoreProjectStyle = context
-                .MSBuildProperties
-                .TryGetValue(ProjectSystemProviderContext.RestoreProjectStyle, out restoreProjectStyle) 
-                && !string.IsNullOrEmpty(restoreProjectStyle);
-
-            var hasTargetFramework = context
-                .MSBuildProperties
-                .TryGetValue(ProjectSystemProviderContext.TargetFramework, out targetFramework)
-                && !string.IsNullOrEmpty(targetFramework);
-
-            var hasTargetFrameworks = context
-                .MSBuildProperties
-                .TryGetValue(ProjectSystemProviderContext.TargetFrameworks, out targetFrameworks)
-                && !string.IsNullOrEmpty(targetFrameworks);
-
-            // check for RestoreProjectStyle property is set and if set to PackageReference then return false
-            if (hasRestoreProjectStyle && !restoreProjectStyle.Equals(ProjectStyle.PackageReference.ToString(), StringComparison.OrdinalIgnoreCase))
+
+            // Check if the project is not CPS capable or if it is CPS capable then it does not have TargetFramework(s), if so then return false
+            if (!hierarchy.IsCapabilityMatch("CPS"))
```

Comment:
I dont wanna add any new capability check at this point in  time and dont want to break anything working. Adding PackageReference capability or RestoreNomination capability can't be done for RTW and has to wait until next release